docs/eigenlayer/operator-guides/operator-installation.md

```diff
@@ -318,10 +317,34 @@ eigenlayer operator status operator.yaml
 
 ---
 
-### Metadata Updates
+## Metadata Updates
 
 This is the command you can use to make changes or updates to the metadata of your operator.
 
 ```
 eigenlayer operator update operator.yaml
 ```
+
+
+
+## delegationApprover Design Patterns
+
+Delegation Approver functionality can be used in multiple ways to give Operators additional programmatic control over which Restakers they accept delegation from.
+
+
+### EOA Signature Approval
```

Comment:
this heading seems inaccurate, especially since smart contract delegationApprovers are one of the main discussion points below. recommend changing the heading to "Passing Signatures from the DelegationApprover to Stakers" -- see my parallel suggestion below for heading